platforms/core-runtime/native/src/main/java/org/gradle/internal/nativeintegration/jansi/JansiLibraryFactory.java

```diff
@@ -16,30 +16,15 @@
 
 package org.gradle.internal.nativeintegration.jansi;
 
-public class JansiLibraryFactory {
-
-    public final static String MAC_OSX_LIB_FILENAME = "libjansi.jnilib";
-    public final static String LINUX_LIB_FILENAME = "libjansi.so";
-    public final static String WINDOWS_LIB_FILENAME = "jansi.dll";
-    private JansiRuntimeResolver jansiRuntimeResolver = new DefaultJansiRuntimeResolver();
-
-    void setJansiRuntimeResolver(JansiRuntimeResolver jansiRuntimeResolver) {
-        this.jansiRuntimeResolver = jansiRuntimeResolver;
-    }
+import org.fusesource.jansi.internal.OSInfo;
```

Comment:
We used to depend on internal details about where libraries are and what they were called, now we depend on an internal class that provides the relevant information.

This makes a bunch of code we had obsolete.